build-rev.sh

```diff
@@ -0,0 +1,33 @@
+#!/usr/bin/env bash
+
+# This command is used by bazel as the workspace_status_command
+# to implement build stamping with git information.
+
+set -o errexit
```

Comment:
all of this can be replaced with `set -euo pipefail`

Reply:
You are right, let me change it as well and nice catch!